adds/start.sh

```diff
@@ -58,7 +58,13 @@ case $OPTION in
 	"start")
 		if [ -f /data/turnserver.conf ]; then
 			echo "-=> start turn"
-			/usr/bin/turnserver --daemon -c /data/turnserver.conf
+			if [ -f /conf/turnserver.conf.deactivated ]; then
+				mv -f /conf/turnserver.conf.deactivated /conf/turnserver.conf
+			fi
+		else
+			if [ -f /conf/turnserver.conf ]; then
+				mv -f /conf/turnserver.conf /conf/turnserver.conf.deactivated
+			fi
```

Comment:
i think its the supervisord config file for turnserver. if there is no `/data/turnserver.conf` then deactivate turnserver in supervisord... and if there is `/data/turnserver.conf` and the turnserver supervisord conf is deactivated... activate it with moving the file.